pkgs/by-name/bp/bpfilter/package.nix

```diff
@@ -0,0 +1,77 @@
+{
+  bison,
+  cmake,
+  cmocka,
+  doxygen,
+  fetchFromGitHub,
+  flex,
+  lcov,
+  lib,
+  libbpf,
+  libnl,
+  nix-update-script,
+  pkg-config,
+  python3,
+  stdenv,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "bpfilter";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "facebook";
+    repo = "bpfilter";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-894SxGhXxkxYD11abl8HfjYyhGFpXGx1JpCsF35/jAw=";
+  };
+
+  nativeBuildInputs = [
+    bison
+    cmake
+    cmocka
+    doxygen
+    flex
+    lcov
+    pkg-config
+    python3.pkgs.sphinx
```

Comment:
I believe there also is top‐level `sphinx` package if the build doesn’t require the library.

Reply:
awesome, switched to use that instead